Library/Homebrew/test/caveats_spec.rb

```diff
@@ -26,4 +27,90 @@ def caveats
       expect(described_class.new(f)).not_to be_empty
     end
   end
+
+  describe "#python_caveats" do
+    subject { described_class.new(f).send(:python_caveats) }
+
+    let(:packages27) { f.lib/"python2.7/site-packages" }
+    let(:packages34) { f.lib/"python3.4/site-packages" }
+
+    before do
+      f.prefix.mkpath
+      f.opt_prefix.parent.mkpath
+      FileUtils.ln_s f.prefix, f.opt_prefix
+    end
+
+    context "python sees our site-packages" do
```

Comment:
I'd prefix all of the `context`s with “when”, or “and” when nested, so it reads like a proper sentence, i.e. “when python sees our site-packages and site-packages is empty, it is expected to be nil”.